tests/fuzzers/bls12381/bls12381_test.go

```diff
@@ -69,9 +69,9 @@ func FuzzG2Add(f *testing.F) {
 	})
 }
 
-func FuzzG2Mul(f *testing.F) {
+func FuzzCrossG2Mul(f *testing.F) {
```

Comment:
```suggestion
func FuzzG2Mul(f *testing.F) {
```

Reply:
I think we covered this in a discussion at some previous point, but we rename here because this now does cross-verification between blst/gnark instead of fuzzing random inputs.

I added random input fuzzers for g1/g2 mul to this PR, because why not.